documentation/configuration/parameters/stateScriptPrefix.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: documentation
+menu: configuration
+pill: stateScriptPrefix
+subtitle: flyway.stateScriptPrefix
+---
+
+# State Script Prefix
+{% include teams.html %}
+
+## Description
+The file name prefix for state scripts.
+
+State scripts represent all migrations with `version <= current state script version` while keeping older migrations if needed for upgrading older deployments.
+
+They have the following file name structure: prefixVERSIONseparatorDESCRIPTIONsuffix, which using the defaults translates to S1.1__My_description.sql
+
+## Default
+S
+
+## Usage
+
+### Commandline
+```powershell
+./flyway -stateScriptPrefix="IB" info
```

Comment:
Do we want to mention 'IB' here?

Reply:
This could be anything, I just went with something familiar to make it easier for the community to adjust since IB is something a lot of them are familiar with